device/arista/x86_64-arista_7060x6_64pe/Arista-7060X6-64PE-256x100G/buffer_ports.j2

```diff
@@ -0,0 +1,6 @@
+{%- macro generate_port_lists(PORT_ALL) %}
```

Comment:
hi Chris, do you mind to update the HWSKU name according to our naming convention, e.g. C256 for 256x100G?

Reply:
Hi Riff, I will update the name.

Can you confirm this result will be sufficient: Arista-7060X6-64PE-C256

Reply:
@r12f we also need to factor in the two 10G SFP ports. That would make the name: `Arista-7060X6-64PE-C256S2`
Please confirm that is acceptable.

Reply:
yea, as discussed offline, the sku name looks great to me!